### PR TITLE
Resources container

### PR DESCRIPTION
**Pull request by Erin007**
This PR renames the GradientNavCard component to ResourceCard, and makes a container component, TeacherResources, to be used on the Teacher Homepage. Visuals are the same, but the components are now better named (should we decide they will no longer have a gradient overlay).

### CODE REVIEW

File: apps/src/templates/teacherHomepage/Resources.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            link="https://code.org/volunteer/local"
+          />
+        </CollapsibleSection>
+      );
+    }
+    if (type === 'student') {
+      return (
+        <CollapsibleSection header={i18n.resources()}>
+          <div>Brendan, you can put student-specific ResourceCards here.</div>
+        </CollapsibleSection>
```
Comment (Erin007) [inline apps/src/templates/teacherHomepage/Resources.jsx]: @breville

**Comment by Erin007** [comment]
cc: @joshlory @tanyaparker

File: apps/src/templates/teacherHomepage/Resources.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import i18n from "@cdo/locale";
+
+const Resources = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(['student', 'teacher']).isRequired
+  },
+
+  render() {
+    const { type } = this.props;
+    if (type === 'teacher'){
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/Resources.jsx]: nit: should have space after close paren

File: apps/src/templates/teacherHomepage/Resources.jsx (hunk 1 above)
Comment (Bjvanminnen) [reply apps/src/templates/teacherHomepage/Resources.jsx]: Should TeacherResources and StudentResources just be two separate components? Or is this something that will be on a page where we can toggle between viewing as teacher vs. student?

Comment (Erin007) [reply apps/src/templates/teacherHomepage/Resources.jsx]: You'll never toggle between viewing as a teacher and a student on the page.  I'll make it just a TeacherResources component and @breville can make StudentResources if he's inspired.

**Pull request merged**